### ISSUE TITLE
Increase max Distance of Douglas-Peuker Algorithm

### ISSUE THREAD

**Issue by pingurus** [Maproom/qmapshack#420]
### Is your feature request related to a problem? Please describe.
[comment]: # (A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] )
I wated to use the new track-to-route feature for the first time, however I can't bring the track under the magical border of 50 trackpoints so my Garmin can do turn by turn instructions on it.

The maximum distance limit the Douglas-Peuker Algorithm which I use for reduction of track points currently allows is 99m. This is not enough for my use-case.

### Describe the solution you'd like
[comment]: # (A clear and concise description of what you want to happen.)
I which this limit would be increased to 1000m.

### Describe alternatives you've considered
[comment]: # (A clear and concise description of any alternative solutions or features you've considered.)
Do the weeding of track-points it by hand

### Give a reason why someone else should spend time on that request
[comment]: # (Keep in mind that you ask others to spend their spare time for your request. Good reasons are if you are able to prove that the request solves a fundamental problem for everyone, or the request is of general interest for many users. Bad reasons are you just want it to have that way, or argue that software xyz does it.)
It shouldn't take too long to change that max number + I'd be willing to do it if there are no reasons against this and someone points me to the place in code.

### Additional context
[comment]: # (Add any other context or screenshots about the feature request here.)

**Comment by kiozen** [comment]
I am currently away with a real bad Internet connectivity. Kind of digital detox.

Concerning the request. A value of 1000 means that a line can deviate 1000m from the wanted line. IMHO at these values results are more or less random. You will be better of doing it manually in that case or have a better algorithm that takes street segments into account.

⁣BlueMail for Android herunterladen ​

**Comment by pingurus** [comment]
Yeah, I doubt too that 1000m is still a useful value, for me something like 110 most probably would have been enough. I simply chose 1000 since that is the next order of magnitude, so we wouldn't have the same issue when next time someone needs 251 when we placed the limit at 250

I think I read once somewhere that Brouter was working on a feature to do track -> waypoints.

Anyways, have a good time on your digital detox!
